06_puppy/nicholascross/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sync"
+)
+
+type Puppy struct {
+	ID     uint64
+	Breed  string
+	Colour string
+	Value  float64
+}
+
+type Storer interface {
+	CreatePuppy(p Puppy)
+	RetrievePuppy(uint64) *Puppy
+	UpdatePuppy(uint64, Puppy)
+	DeletePuppy(uint64)
+}
```

Comment:
None of your interface functions return errors. So how does function caller know something is wrong? For example, update or delete with non-existing id?

Reply:
This is working as intended, I have already implemented error handling in the subsequent lab.  I don't want to double up on work so unless this is going to be a blocker I will leave it as is.